src/redux/books/books.js

```diff
@@ -0,0 +1,34 @@
+const ADD = 'ADD_BOOK';
+const REMOVE = 'REMOVE_BOOK';
+
+const booksXer = (state = [{ id: '1', title: 'Elden Ring', author: 'Miyazaki' }, { id: '2', title: 'One Piece', author: 'Eichiro Oda' }], action) => {
```

Comment:
- [ ] Initial states should be defined as a stand-alone variables. This is good for situations like this where you want to initialize your state to something. It gets messy doing so in the reducer arguments as you can see.

  Kindly define an initial state and set the state to the initial state.
  ```
  const initialState = []
  ...
  const booksXer = (state = initialState, action) => {
    ...
  ```